### PR TITLE
Updated README.md with examples of access token as parameter

### PR DESCRIPTION
**Pull request by khendrikse**
Updated README.md with examples of access token as parameter

### CODE REVIEW

**Comment by pradel** [comment]
that's great thanks! can you please rebase? :)

**Comment by khendrikse** [comment]
Hopefully this works! Sorry, it took me some time to figure it out. :)

File: README.md [hunk 1]
```diff
@@ -184,9 +184,24 @@ instagram.get('tags/:tag-name/media/recent').then(data => {
...
+It is also possible to send the access_token along as a parameter when you call an endpoint. For example:
+
+```javascript
+// Get information about current user
+instagram.get('users/self', { access_token: accessToken }, (err, data) => {
+  console.log(data);
+});
+
+// Search for tags by name.
+instagram.get('tags/search, { access_token: accessToken, 'q: 'paris' }).then((data) => {
```
Comment (pradel) [inline README.md]: Just a little typo here :)
`'tags/search, { access_token: accessToken, 'q: 'paris' }` have wrong apostrophe placement.

File: README.md [hunk 2]
```diff
@@ -184,9 +184,24 @@ instagram.get('tags/:tag-name/media/recent').then(data => {
 });
 
 // Search for tags by name.
-instagram.get('tags/search', { q: 'paris' }).then(data => {
+instagram.get('tags/search, {'q: 'paris' }).then((data) => {
```
Comment (pradel) [inline README.md]: `'tags/search, {'q: 'paris' }` have wrong apostrophe placement.

**Comment by pradel** [comment]
Also can you please run `npm run format` before the commit (it will make the ci green :) )

**Comment by khendrikse** [comment]
Auch sorry for that, that was very messy of me. Hopefully now it works right?

**Comment by pradel** [comment]
Thats perfect thank you for the pr! :)

**Pull request merged**